### ISSUE TITLE
Meeting Times / day of week / # of meetings per month ? 

### ISSUE THREAD

**Issue by a-schaefers** [cooslug/cooslug.org#3]
What is the best time and day of the week and number of meetings to have per month?

**Comment by a-schaefers** [comment]
I want to suggest an informal weekly meeting and also  presentation-style monthly meeting.

This might be too much at first though, perhaps bi-monthly would be ideal, with the meeting in the first week of the month being the presentation.

**Comment by bayprogrammer** [comment]
Weekly meetings may be too frequent. I know I wouldn't be able to attend them weekly myself. Bi-monthly presentations might be a good starting point, building up from there as our ranks expand.

**Comment by a-schaefers** [comment]
Yep, I agree. I am kinda going off of http://euglug.org/ , their page says what they do is as follows:

> **Monthly Presentations**
> 
> Once a month we have a presentation on a technical topic. These are generally held on the first Tuesday of the month, at 6 PM. For details, see the Meetup page. RSVP for the next meeting. 
> 
> **Weekly Meetings**
> 
> We meet on Thursday evenings at 7 PM. These are highly informal, laid-back affairs, and a good place to come with Linux-related questions and pleas for help.
> 
> To find out the current meeting location, send email to info@euglug.org.

**Comment by a-schaefers** [comment]
I'm worried if the meetings are too far apart, they may not "stick" in people's minds, but if they are too close together, it could become burdensome for all of the faithful.

**Comment by a-schaefers** [comment]
I like having one solid, presentation meeting every month. And more "highly informal" hopefully, may be added on a biweek. And then if it comes to that people want more than 2 meets a month, we can just add more and keep going with it.
